### PR TITLE
rping: Add config to bind to source IP address like ping.

### PR DESCRIPTION
**Pull request by rldleblanc**
This is useful if you have multiple adapters on the same
network or have custom routing that you want to test.

Signed-off-by: Robert LeBlanc <robert@leblancnet.us>

### CODE REVIEW

File: librdmacm/examples/rping.c [hunk 1]
```diff
@@ -1143,6 +1149,7 @@ static void usage(const char *name)
 	printf("%s -c [-vVd] [-S size] [-C count] -a addr [-p port]\n", 
 	       basename(name));
```
Comment (hnrose) [inline librdmacm/examples/rping.c]: Should add [-I src_addr].

Also, should update rping.1 man page.

**Comment by hnrose** [comment]
@shefty

**Comment by shefty** [comment]
Looks good to me.  Steve Wise wrote rping, I don't know what his github id is.

**Comment by larrystevenwise** [comment]
larrystevenwise :)

**Pull request merged**